.github/workflows/performance.yaml

```diff
@@ -120,14 +97,75 @@ jobs:
           sudo mksquashfs snap-unpack-dir v1-2-0.snap -noappend -comp lzo -no-fragments
       - name: Switch back to target branch
         run: git reset --hard $TARGET_SHA
+      - name: Run Performance test ${{ github.head_ref }} snap
+        env:
+          TEST_SNAP: ${{ github.workspace }}/head.snap
+          TEST_SUBSTRATE: lxd
+          TEST_LXD_IMAGE: ubuntu:22.04
+          TEST_INSPECTION_REPORTS_DIR: ${{ github.workspace }}/inspection-reports
+          TEST_METRICS_DIR: ${{ github.workspace }}/test/performance/results/head
+          TEST_RUN_NAME: head
+        run: |
+          cd test/performance
+          mkdir -p ./results/head
+          sg lxd -c 'tox -e performance'
+      - name: Run Performance test for base code snap
+        env:
+          TEST_SNAP: ${{ github.workspace }}/base-code.snap
+          TEST_SUBSTRATE: lxd
+          TEST_LXD_IMAGE: ubuntu:22.04
+          TEST_INSPECTION_REPORTS_DIR: ${{ github.workspace }}/inspection-reports
+          TEST_METRICS_DIR: ${{ github.workspace }}/test/performance/results/base-code
+          TEST_RUN_NAME: base-code
+        run: |
+          cd test/performance 
+          mkdir -p ./results/base-code
+          sg lxd -c 'tox -e performance'
+      - name: Run Performance test for v1.1.11 snap
```

Comment:
Out of curiosity, why are you specifically testing v1.1.11? What's special about this version?

Reply:
The v1.1.11 is the tag before we introduced the compaction changes (v1.2.0).
So we can compare where we are now with v1.11.1:

![image](https://github.com/user-attachments/assets/812036ff-1bd1-43b7-ae04-3cf678e23241)